src/modules/previewpane/common/handlers/PreviewHandlerBase.cs

```diff
@@ -201,7 +203,7 @@ private static void RegisterPreviewHandler(string name, string extensions, strin
             }
 
             // Add preview handler to preview handler list
-            using (RegistryKey handlersKey = Registry.CurrentUser.OpenSubKey("SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\PreviewHandlers", true))
+            using (RegistryKey handlersKey = Registry.LocalMachine.OpenSubKey("SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\PreviewHandlers", true))

```

Comment:
why the change?

Reply:
This was a bug. `RegisterPreviewHandler` function uses Local machine registry key.